packages/web/src/pages/upload-page/store/utils/stems.ts

```diff
@@ -36,9 +24,9 @@ export const createStemMetadata = ({
   stemCategory
 }: {
   parentTrackId: ID
-  track: Track
+  track: TrackForUpload['metadata']
```

Comment:
I'd love to make a habit of being careful what we name our track-like parameters. I see `track` and never know what it means without referencing the arg type. Maybe we can start naming these `trackMetadata`?

Reply:
better yet, one definitive model for a track 😢 I think most cases track metadata === track, it's just weird in upload and edit scenarios